From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Wed, 19 Aug 2020 04:42:53 +0000
X-Dgit-Generated: 8.0-3~0~20200819~1 5b649367b94d7c2e60e36e29b18019169a9a75e6
Subject: Remove USE_SYSTEMD guards around several scripts that should always

get installed.

The build-time detection of systemd support only appears to work when
usrmerge is enabled, resulting in different packages on usrmerge
vs. non-usrmerge systems, breaking reproducible builds.

---

--- glusterfs-8.0.orig/extras/Makefile.am
+++ glusterfs-8.0/extras/Makefile.am
@@ -2,10 +2,8 @@ addonexecdir = $(GLUSTERFS_LIBEXECDIR)
 addonexec_SCRIPTS =
 if WITH_SERVER
 addonexec_SCRIPTS += peer_add_secret_pub
-if USE_SYSTEMD
 addonexec_SCRIPTS += mount-shared-storage.sh
 endif
-endif
 
 EditorModedir = $(docdir)
 EditorMode_DATA = glusterfs-mode.el glusterfs.vim
@@ -33,11 +31,9 @@ scripts_SCRIPTS = thin-arbiter/setup-thi
 if WITH_SERVER
 scripts_SCRIPTS += post-upgrade-script-for-quota.sh \
 	pre-upgrade-script-for-quota.sh stop-all-gluster-processes.sh
-if USE_SYSTEMD
 scripts_SCRIPTS += control-cpu-load.sh
 scripts_SCRIPTS += control-mem.sh
 endif
-endif
 
 EXTRA_DIST = glusterfs-logrotate gluster-rsyslog-7.2.conf gluster-rsyslog-5.8.conf \
 	logger.conf.example glusterfs-georep-logrotate group-virt.example \
